templates/common/_base/files/kubelet-auto-sizing.yaml

```diff
@@ -0,0 +1,99 @@
+mode: 0755
+path: "/usr/local/sbin/dynamic-system-reserved-calc.sh"
+contents:
+  inline: |
+    #!/bin/bash
+    set -e
+    NODE_SIZES_ENV=${NODE_SIZES_ENV:-/etc/node-sizing.env}
+    function dynamic_memory_sizing {
```

Comment:
What these values are based on? Not implying they're wrong, but I'm curious to see profiling data and/or graphs, if any.